src/screens/Sidebar.svelte

```diff
@@ -18,11 +19,23 @@
 	import settings from "../assets/settings.svg";
 	import logout from "../assets/logout.svg";
 	import groupcat from "../assets/meowy.svg";
+	import gc from "../assets/chat.svg"
 
 	/**
 	* @param {any} newPage Goes to a page while also refreshing it.
 	*/
 	function goto(newPage) {
+		cljs.send({
```

Comment:
uhh, don't send a request to leave the group chat everytime a page is switched, you should check if the last page was a group chat or not

Reply:
uhhh how could i do that quickly

Reply:
> uhhh how could i do that quickly

https://github.com/meower-media-co/Meower-Svelte/pull/20